startup_gateway/chat/middleware.py

```
@@ -8,33 +8,45 @@
+            user = User.objects.get(id=user_id)
+
+            if user.jwt_version != token_version:
+                return AnonymousUser()
+
+            return user
+
+        except jwt.ExpiredSignatureError:
+            return AnonymousUser()
+        except jwt.InvalidTokenError:
+            return AnonymousUser()
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)